recipe_scrapers/_utils.py

```diff
@@ -230,6 +230,18 @@ def normalize_string(string):
     )
 
 
+def csv_to_tags(csv):
+    raw_tags = csv.split(",")
+    seen = set()
+    tags = []
+    for raw_tag in raw_tags:
+        tag = raw_tag.strip().lower()
+        if tag not in seen:
+            seen.add(tag)
+            tags.append(tag)
+    return tags
```

Comment:
Looking at some of the test data for the keywords, I'm having second thoughts about whether we always want to lowercase each tag.

`mittagessen` should generally be capitalized I think, for example - and I can imagine country names appearing in keywords sometimes (similar to cuisine).

Keeping the de-duplication does still make sense though I think?

Current idea:

```suggestion
def csv_to_tags(csv, lowercase=False):
    raw_tags = csv.split(",")
    seen = set()
    tags = []
    for raw_tag in raw_tags:
        tag = raw_tag.strip()
        if tag.lower() not in seen:
            seen.add(tag.lower())
            tags.append(tag.lower() if lowercase else tag)
    return tags
```

Maybe that's overcomplicating things; it would only emit one entry, `[Tags]`, from an input of `Tags, tags`, by default.